bin/gltf-pipeline.js

```diff
@@ -39,29 +41,35 @@ fs.readFile(gltfPath, function (err, data) {
     }
 
     var gltf = JSON.parse(data);
-    var stats = new OptimizationStatistics();
+    gltf = loadGltfUris(gltf, path.dirname(gltfPath), function(loadError) {
```

Comment:
Sorry, didn't see that recent comment.  Yes, I saw that there is `err` in an outer scope and in another inner scope.  In this particular case, I would not worry about it; the current scope overrides, of course.